src/MathJaxRenderer.jl

```diff
@@ -126,29 +126,31 @@ function svg_converter(
     return IOBuffer(extract_svg(str))
 end
 
+
 function converter(
     f::Math,
     m::SUPPORTED_MIMES;
     dpi_x = 90,
     dpi_y = 90,
     x_zoom = 1.0,
     y_zoom = 1.0,
-    zoom = 1.0,
     keep_aspect_ratio = false,
     background_color = :white,
     unlimited = false,
+    rsvg_convert_bin = "",
```

Comment:
Can we use `nothing` to represent that absence of a provided binary here rather than `""`? Just makes it slightly more explicit within the code that the binary wasn't provided rather than a user providing an empty string themselves.